From 624adf0125b180c5327330ef10079410e1c48a1e Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Thu, 30 Apr 2026 13:15:01 -0500
Subject: [PATCH] Allow absolute RPATH to be set on vendored executables

---
 rpm/ros-lyrical-ortools-vendor.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rpm/ros-lyrical-ortools-vendor.spec b/rpm/ros-lyrical-ortools-vendor.spec
index bb482c7ca92e..2bbcef472b9d 100644
--- a/rpm/ros-lyrical-ortools-vendor.spec
+++ b/rpm/ros-lyrical-ortools-vendor.spec
@@ -14,6 +14,8 @@ Source0:        %{name}-%{version}.tar.gz
 BuildRequires:  bloom-rpm-macros
 BuildRequires:  cmake
 
+%global __brp_check_rpaths QA_RPATHS=0x0002 %{__brp_check_rpaths}
+
 %{?bloom_package}
 
 %description
-- 
2.53.0

